### DISCUSSION TITLE
What is the formula of the accuracy in osu!mania on earth?

### DISCUSSION THREAD

**Discussion by Nana-Sakura** [ppy/osu#22413] [Q&A] [answered]
I have checked the wiki and found that the formula of accuracy for Score V1 is
[N(Max+300)\*300+N(200)\*200+N(100)\*100+N(50)\*50]/300\*N(All Notes)
Score V2 is
[N(Max)\*305+N(300)\*300+N(200)\*200+N(100)\*100+N(50)\*50]/305\*N(All Notes)
But a Great in osu!mania on osu!(Lazer), corresponding with 300 in Classic, has the accuracy of 95.23%, lower than 300/305~98.36%. Looks like 300/315.
Is it a bug, or it ought to be?

~~(Edit): 
Btw, how can I export the osu! argon skin, making it available for Classic Client? 
I tried to export the modified one but the file is sized only around 400 Bytes.
(I like the latest skin and wanted to make some changes but I could not)~~

**Comment by mk56-spn** [comment]
The argon skin can not be exported for use elsewhere. It is not a typical osk . It is hard coded with classes that define it's entire behaviour pretty much. You can find imitation skins online . But they can't match the sliders, or a bunch of other things

**Comment by bdach** [comment, marked as answer]
Accuracy is not a bug. Read https://github.com/ppy/osu/wiki/Scoring.
